app/src/main/java/com/qsp/player/game/MainActivity.java

```diff
@@ -484,9 +486,14 @@ private void addSaveSlotsSubMenu(MenuItem parent, final SlotAction action) {
         for (int i = 0; i < MAX_SAVE_SLOTS; ++i) {
             final String filename = getSaveSlotFilename(i);
             final DocumentFile file = savesDir.findFile(filename);
-            String title = String.format(
-                    getString(file != null ? R.string.slotPresent : R.string.slotEmpty),
-                    i + 1);
+            String title = null;
+
+            if (file != null) {
+                final String lastMod = (String) DateFormat.format("yyyy-MM-dd kk:mm:ss", file.lastModified());
+                title = String.format(getString(R.string.slotPresent), i + 1, lastMod);
```

Comment:
Can be simplified to `getString(R.string.slotPresent, i + 1, lastMod)`. See https://developer.android.com/reference/android/content/Context#getString(int,%20java.lang.Object...)